MAINTAINER.md

```diff
@@ -0,0 +1,334 @@
+      if you would like to demonstrate a solution, please open a PR :smiley: ."
+
+    - "Hi @\<op-name\>! Thanks for creating a help ticket. I will do some research and get
+      back to you asap!"
+
+
+**Notes:**
+
+Try not to forget about issues. If you don't know what they are asking for,
+don't understand the bug, or don't even know if it is a thing we should care about,
+tag in the rest of the team on slack after you have written your boilerplate "Thanks for opening"
+message.
+
+
+## Plan for a sprint
+
+The core team works in weekly sprints and gathers every Wednesday to decide
+what their focus will be for the coming week.
+
+Whoever is [on call](#on-call) both decides what the core team will be working on and runs the
+planning meeting. _The power..._ :astonished: :star_struck: .
+
+### Pre-planning
+
+This involves moving tickets around in the `eksctl-backlog` and ensuring there
+is a prioritized set in `Ready for work`.
+
+Depending on the complexity of tickets, we want to end up with from 5-10 issues
+teed up in `Ready for work`.
+
+- Check in with AWS to see if there are any new features coming out, check the [container
+  roadmap](https://github.com/aws/containers-roadmap) for new things coming out and create mirror issues in eksctl.
+- Check the `Backlog` column in `eksctl-backlog` to see if anyone has added anything
+  good. Move tickets you want into `Ready for work`.
+- Check the issues. `priority/critical` issues should already be in the `Sprint In Progess`
+  column (or at least in `Ready for work`), but if there are some which haven't been scheduled
+  move them into ready.
+- Go through other `priority/<x>` issues and move some which look ready for
+  work (ie. have sufficient context) into that column.
+- Ping the core team to ask if they have anything they would like to prioritize.
+- Prioritize the issues into the order you think they should be tackled.
+
+### Running the meeting
+
+Go through each ticket in `Ready for work`, including the ones which are still there
+from last sprint. For each one:
+- Remind everyone of the context
+- Explain why you prioritized it for this week
+- Explain the user value it would deliver (if applicable)
+- Ask if the scope is clear, if it is ready for work, or if folks feel they need more
+- Give space for discussion and questions
+
+## Create a release
+
+1. Check out to master, make sure the branch is up to date.
+1. Ensure [integration tests](CONTRIBUTING.md#running-the-integration-tests) pass (est: 3 hours).
+1. Make a PR to change the API version (`v1alpha5`) if necessary.
+1. Determine the next release tag, e.g.:
+
+   - for a release candidate, `0.13.0-rc.0`, or
+   - for a release, `0.13.0`.
+
+1. Create a `docs/release_notes/<version>.md` release notes file for the given tag using the contents of the release
+   draft (generated by [`release-drafter`](https://github.com/release-drafter/release-drafter)), e.g.:
```

Comment:
where does one find this again? I remember not knowing where it was during my last release before the holidays

Reply:
in the release draft on github

Reply:
can we add that to the doc?

Reply:
is that not already what it says? i paraphrased the line you highlighted

Reply:
oh yeah re-reading it that makes sense. I sort of merged the words "release draft" and "release-drafter" into one in my head 🤦